docs/pairing.puml

```diff
@@ -0,0 +1,20 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+title Pairing
+
+autonumber
+actor User
+
+User -> Device: Boot k4e device ISO
+Device -> Agent: Start agent
+Agent -> Operator: Send pairing request
+Operator -> "EdgeDevice CR": Create
+Operator -> Agent: Pairing confirmation
+Operator -> OBC: Create Object Bucket Claim for the device data
+Operator -> "EdgeDevice CR": Update status with OBC info
+Agent -> Agent: Start processing requests
+Agent -> Clock: Start data directories monitoring
```

Comment:
What does 'Clock' represents ?

Reply:
Just an abstract clock (timer) that triggers some periodic actions.